frontend/src/metabase/dashboard/components/DashCard/DashCard.tsx

```diff
@@ -257,7 +257,7 @@ function DashCardInner({
     <ErrorBoundary>
       <DashCardRoot
         data-testid="dashcard"
-        className="Card rounded flex flex-column hover-parent hover--visibility"
+        className="Card relative rounded flex flex-column hover-parent hover--visibility"
```

Comment:
Why could we remove `relative` here? Is it because the component already have `position: relative` applied?